Content.Server/BluespaceHarvester/BluespaceHarvesterRiftComponent.cs

```diff
@@ -0,0 +1,34 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.List;
+
+namespace Content.Server.BluespaceHarvester;
+
+[RegisterComponent]
+public sealed partial class BluespaceHarvesterRiftComponent : Component
+{
+    [DataField("danger"), ViewVariables(VVAccess.ReadWrite)]
+    public int Danger = 0;
+
+    [DataField("passiveSpawnAccumulator"), ViewVariables(VVAccess.ReadWrite)]
+    public float PassiveSpawnAccumulator = 30f;
+
+    [DataField("passiveSpawnCooldown"), ViewVariables(VVAccess.ReadWrite)]
+    public float PassiveSpawnCooldown = 30f;
+
+    [DataField("passiveSpawn", customTypeSerializer: typeof(PrototypeIdListSerializer<EntityPrototype>)), ViewVariables(VVAccess.ReadWrite)]
+    public List<string> PassiveSpawnPrototypes = new();
+
+    [DataField("spawn")]
+    public List<EntitySpawn> SpawnPrototypes = new();
+}
+
+[Serializable, DataDefinition]
+public partial struct EntitySpawn
+{
+    [DataField("id", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>)), ViewVariables(VVAccess.ReadWrite)]
+    public string? PrototypeId = null;
+
+    [DataField("cost"), ViewVariables(VVAccess.ReadWrite)]
+    public int Cost = 1;
```

Comment:
```suggestion
    [DataField, ViewVariables(VVAccess.ReadWrite)]
    public int Danger = 0;

    [DataField, ViewVariables(VVAccess.ReadWrite)]
    public float PassiveSpawnAccumulator = 30f;

    [DataField ViewVariables(VVAccess.ReadWrite)]
    public float PassiveSpawnCooldown = 30f;

    [DataField("passiveSpawn", customTypeSerializer: typeof(PrototypeIdListSerializer<EntityPrototype>)), ViewVariables(VVAccess.ReadWrite)]
    public List<string> PassiveSpawnPrototypes = new();

    [DataField("spawn")]
    public List<EntitySpawn> SpawnPrototypes = new();
}

[Serializable, DataDefinition]
public partial struct EntitySpawn
{
    [DataField("id", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>)), ViewVariables(VVAccess.ReadWrite)]
    public string? PrototypeId = null;

    [DataField, ViewVariables(VVAccess.ReadWrite)]
    public int Cost = 1;
```

Reply:
Yes, now it really is in the RnD study branch and its board is created there.
![image](https://github.com/space-wizards/space-station-14/assets/54727692/90c1630e-c22f-4eb5-9c45-a705d69847f8)
![image](https://github.com/space-wizards/space-station-14/assets/54727692/a0a25ca7-602a-4581-ac03-a19adc56a2ec)